kubenow/gluster-s3/values.yaml

```diff
@@ -0,0 +1,19 @@
+# Default values for a Gluster-S3 deployment.
+# this is a yaml-formatted file.
+# declare variables to be passed into your templates.
+releaseName: s3-interface
+secretName: s3-secret
+
+account: account
+user: user
+password: 8VBdiwxL2s
```

Comment:
never hardcode passwords, default to null

Reply:
not even as default values?

Reply:
i have read the best practices and looked at official charts and they do this